packages/db/src/lib/granule.ts

```diff
@@ -354,3 +357,39 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
+ *
+ * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+ * @param {Array<ApiGranule>} [granules] - updated ApiGranule records
+ * @returns {Promise<void>}
+ */
+export const updateGranuleAndFiles = async (
```

Comment:
This takes an input array of multiples, it should probably be `updateGranulesAndFiles`